code/addons/controls/package.json

```diff
@@ -52,13 +53,16 @@
   },
   "dependencies": {
     "@storybook/blocks": "workspace:*",
+    "@storybook/core-common": "workspace:*",
+    "cjs-module-lexer": "^1.2.3",
+    "es-module-lexer": "^1.5.0",
+    "globby": "^14.0.1",
```

Comment:
`glob` itself lists `globby` as an alternative when e.g. `.gitignore` should be taken into account. We could, of course, use `globby` instead of `glob` everywhere, but this refactoring should happen outside of this PR.

Reply:
I wouldn't prebundle that much as long as we have this massive amount of separate mono repo packages and it isn't necessary. `cjs-module-lexer` and `es-module-lexer` are used in three different modules. Prebundling them would unnecessarily increase download size. `globby` is also used many times. I wouldn't prebundle it either.